package.json

```diff
@@ -1,49 +1,98 @@
+  "author": "",
+  "license": "MIT",
+  "repository": "/network-graph",
+  "main": "dist/index.js",
+  "module": "dist/index.modern.js",
+  "source": "src/index.tsx",
+  "engines": {
+    "node": ">=10"
+  },
   "scripts": {
-    "start": "webpack serve --config webpack.config.js --mode development",
-    "build": "webpack --config webpack.config.js --mode production",
+    "build": "microbundle-crl --no-compress --format modern,cjs",
+    "start": "microbundle-crl watch --no-compress --format modern,cjs",
+    "prepare": "run-s build",
     "test": "jest --config jestconfig.json --watch",
     "test:ci": "jest --config jestconfig.json",
     "test:coverage": "jest --config jestconfig.json --coverage",
     "lint": "eslint \"{**/*,*}.{js,ts,jsx,tsx}\" --fix",
+    "predeploy": "cd example && yarn install && yarn run build",
     "format": "prettier --write \"{src,tests,example/src}/**/*.{js,ts,jsx,tsx}\"",
+    "deploy": "gh-pages -d example/build",
     "typecheck": "tsc --noEmit --declaration false"
   },
   "dependencies": {
+    "babel-jest": "^29.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-scripts": "^5.0.1",
     "typescript": "^5.0.4"
   },
   "devDependencies": {
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "^14.4.3",
+    "@types/d3-drag": "^3.0.2",
+    "@types/d3-force": "^3.0.3",
+    "@types/d3-selection": "^3.0.3",
     "@types/jest": "^29.5.1",
-    "@types/react": "^18.2.0",
-    "@types/react-dom": "^18.2.1",
-    "@types/testing-library__jest-dom": "^5.14.5",
-    "@typescript-eslint/eslint-plugin": "^5.59.2",
-    "@typescript-eslint/parser": "^5.59.2",
+    "@types/node": "^20.2.3",
+    "@types/react": "^18.2.7",
+    "@types/react-dom": "^18.2.4",
+    "@typescript-eslint/eslint-plugin": "^5.59.7",
+    "@typescript-eslint/parser": "^5.59.7",
+    "@visx/responsive": "^3.0.0",
+    "@visx/zoom": "^3.1.1",
+    "babel-core": "7.0.0-bridge.0",
+    "babel-eslint": "^10.0.3",
+    "cross-env": "^7.0.2",
     "css-loader": "^6.7.3",
-    "eslint": "^8.39.0",
+    "d3-drag": "^3.0.0",
```

Comment:
let's move these d3-packages to `dependencies`

Reply:
Done